### DISCUSSION TITLE
I bump wit problem "SSLException: No PSK available"

### DISCUSSION THREAD

**Discussion by FedyaSavchuk** [mockoon/mockoon#523] [Help] [answered]
This problem appear when I make async requests in mockoon

**Comment by 255kb** [comment, marked as answer]
I have no idea where it comes from, but after googling a bit it seems to be a JDK problem. 
https://stackoverflow.com/a/54280095/4287789
Have you tried one of the proposed solution there?

**Reply by FedyaSavchuk** [reply]
Yes! It's problem with Java version

**Reply by 255kb** [reply]
Glad you solved it!
